### ISSUE TITLE
Is development still occuring?

### ISSUE THREAD

**Issue by gkaissis** [karolzak/keras-unet#2]
I think this package is great! Is it still under active development? If yes, my team and I would like to contribute, as we have recently started using it for our work.

**Comment by karolzak** [comment]
Hi @gkaissis! 
Flattered to hear you enjoy it :)
This project is definitely open for any contributions! It's still under development however since I've been very busy with other projects last couple months I didn't manage to push any changes so I wouldn't exactly call it an "active development".
Bear in mind that the quality of that code is not that great yet.. for current version I just gathered some of the snippets I found useful. So I'm thinking about refactoring it, adding tests, etc.
Do you have any specific features on your mind you would like to add to the package?

**Comment by gkaissis** [comment]
I think we could benefit from adding more loss functions such as combinations of dice loss and pixelwise losses. Also, I'm not sure whether using batch normalization causes the model to not use biases? I'm in the middle of a lot of work as well at the moment, but I will open a PR as soon as I have time.
